src/NUnitTestAdapter/TestConverter.cs

```diff
@@ -40,61 +40,66 @@ public TestConverter(TestLogger logger, string sourceAssembly)
         /// using the best method available according to the exact
         /// type passed and caching results for efficiency.
         /// </summary>
-        public TestCase ConvertTestCase(ITest test)
+        public TestCase ConvertTestCase(XmlNode testNode)
         {
-            if (test.IsSuite)
+            if (testNode.Name != "test-case")
```

Comment:
Except in our tests, we create the test nodes from an xml string that comes from the driver, so it has to be valid or an exception would have been thrown before we got here.